src/Faker/Provider/dv_MV/Text.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Faker\Provider\dv_MV;
+
+
+class Text extends \Faker\Provider\Text
+{
+    protected static function validStart($word)
+    {
+        return preg_match('/^[ހށނރބޅކއވމފދތލގޏސޑޒޓޔޕޖޗ]/', $word);
+    }
+    
+    protected static $baseText = <<<'EOT'
```

Comment:
Do we have a valid source?

Reply:
ya. 
Ministry of Education - https://www.moe.gov.mv/assets/upload/Dhivehi_1_Dharivarunge_Foiy_UNIT_1_3_2.2.2015.pdf and
Dhivehi Bahuge Academy (Maldivian Goverment body responsible with language related stuff) - http://dhivehiacademy.edu.mv/beta/lang/138

Reply:
got it. updated with source. thank you :)